ui.content/src/main/content/jcr_root/conf/global/settings/workflow/models/hclecomm/content-approval-workflow/.content.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        cq:designPath="/libs/settings/wcm/designs/default"
+        cq:lastModified="{Date}2020-06-16T19:01:59.971+05:30"
+        cq:lastModifiedBy="admin"
+        cq:template="/libs/cq/workflow/templates/model"
+        jcr:primaryType="cq:PageContent"
+        jcr:title="Content Approval Workflow"
+        sling:resourceType="cq/workflow/components/pages/model">
+        <flow
+            jcr:primaryType="nt:unstructured"
+            sling:resourceType="foundation/components/parsys">
+            <participant
+                jcr:created="{Date}2020-06-16T18:28:03.800+05:30"
+                jcr:createdBy="admin"
+                jcr:description="Validate Content"
+                jcr:lastModified="{Date}2020-06-16T18:28:42.515+05:30"
+                jcr:lastModifiedBy="admin"
+                jcr:primaryType="nt:unstructured"
+                jcr:title="Validate Content"
+                sling:resourceType="cq/workflow/components/model/participant">
+                <metaData
+                    jcr:primaryType="nt:unstructured"
+                    PARTICIPANT="content-approvers"/>
+            </participant>
+            <or
+                jcr:created="{Date}2020-06-16T18:28:11.341+05:30"
+                jcr:createdBy="admin"
+                jcr:lastModified="{Date}2020-06-16T19:01:59.342+05:30"
+                jcr:lastModifiedBy="admin"
+                jcr:primaryType="nt:unstructured"
+                sling:resourceType="cq/workflow/components/model/or"
+                branches="2"
+                branchName1="Branch 1"
+                branchName2="Branch 2"
+                branchOrder="[0,1]"
+                default1="true"
+                orSplit="true">
+                <_x0031_
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="cq/flow/components/parsys">
+                    <participant
+                        jcr:created="{Date}2020-06-16T18:28:16.742+05:30"
+                        jcr:createdBy="admin"
+                        jcr:description="Rejecting the content as the content is not valid"
+                        jcr:lastModified="{Date}2020-06-16T18:29:21.024+05:30"
+                        jcr:lastModifiedBy="admin"
+                        jcr:primaryType="nt:unstructured"
+                        jcr:title="Reject the content"
+                        sling:resourceType="cq/workflow/components/model/participant">
+                        <metaData
+                            jcr:primaryType="nt:unstructured"
+                            PARTICIPANT="content-author"/>
```

Comment:
@nitheshkumar-hcl   updated group name and append HCLecomm in name.

Reply:
updated the workflow as per the review comments